src/logging.cc

```diff
@@ -1459,7 +1459,7 @@ static void logging_fail() {
 #if defined(_DEBUG) && defined(_MSC_VER)
   // When debugging on windows, avoid the obnoxious dialog and make
   // it possible to continue past a LOG(FATAL) in the debugger
-  _asm int 3
+  __debugbreak();
```

Comment:
no need to include <intrin.h> ?
https://msdn.microsoft.com/en-us/library/f408b4et.aspx